index.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
   extends: ['tslint:latest', 'tslint-config-prettier'],
   rules: {
+    'await-promise': true,
     'interface-name': [true, 'never-prefix'],
     'member-access': false,
+    'no-async-without-await': true,
     'no-floating-promises': true,
```

Comment:
This reminds me that `no-floating-promises` clashes with async `useEffect` hooks 🤔